docs/course_materials/exercises/rose_check.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import argparse
+import importlib
+import logging
+import os.path
+from pathlib import Path
+import sys
+import linux_tester
+
+
+def load_file(file_path):
+    """
+    Load the file as module
+
+    Arguments:
+        file_path (str): The path to the file
+
+    Returns:
+        File as module
+
+    Raises:
+        FileNotFoundError if the file cannot be loaded
+    """
+    module_name = os.path.split(os.path.splitext(file_path)[0])[1]
+    module_spec = importlib.util.spec_from_file_location(module_name, file_path)
+    if module_spec is None:
+        print('Module: {} not found'.format(module_name))
+        return None
+    else:
+        try:
+            module = importlib.util.module_from_spec(module_spec)
+            module_spec.loader.exec_module(module)
+            return module
+        except FileNotFoundError as e:
+            print("Error loading the file %r: %s" % (file_path, e.strerror))
```

Comment:
Two things about string formatting:
1. Since we're using Python 3, let's use f-strings :) You're using C-style formatting here and `format` method above.
2. We should be consistent with quotes. You've used single quotes previously, so let's stick to that.